src/pages/ResearcherConsole.js

```diff
@@ -152,17 +112,27 @@ export default function ResearcherConsole() {
     }
   };
 
+
+  //Draft delete, by referenceIds
+  const deleteDraftById = async ({ referenceId, collectionsClone }) => {
+    await DAR.deleteDar(referenceId);
+    const targetIndex = findIndex((draft) => {
+      return draft.referenceId === referenceId;
```

Comment:
I think this line should either be `draft.referenceIds[0] === referenceId` or `includes(referenceId, draft.referenceIds) ` since summaries have the `referenceIds` array

Reply:
good catch